protocol/messenger_handler.go

```diff
@@ -1712,3 +1713,61 @@ func (m *Messenger) updateUnviewedCounts(chat *Chat, mentioned bool) {
 		chat.UnviewedMentionsCount++
 	}
 }
+
+func (m *Messenger) HandleSyncWalletAccount(state *ReceivedMessageState, message protobuf.SyncWalletAccounts) error {
```

Comment:
Do we need to also send data to the account changes feed?

Reply:
What is that account changes feed? What is it used for? (pardon my ignorance)